modules/data-streams/src/main/java/org/elasticsearch/datastreams/rest/RestGetDataStreamsAction.java

```diff
@@ -57,4 +58,19 @@ public boolean allowSystemIndexAccessByDefault() {
     public Set<String> supportedCapabilities() {
         return Set.of(DataStreamLifecycle.EFFECTIVE_RETENTION_REST_API_CAPABILITY);
     }
+
+    @Override
+    public Set<String> supportedQueryParameters() {
+        return Set.of(
+            "name",
+            "include_defaults",
+            "timeout",
+            "master_timeout",
+            IndicesOptions.WildcardOptions.EXPAND_WILDCARDS,
+            IndicesOptions.ConcreteTargetOptions.IGNORE_UNAVAILABLE,
+            IndicesOptions.WildcardOptions.ALLOW_NO_INDICES,
+            IndicesOptions.GatekeeperOptions.IGNORE_THROTTLED,
+            "verbose"
```

Comment:
We ought to add this to get-data-stream.asciidoc, right?

Reply:
Yes good point, I'll add it to the docs

Reply:
Okay, I added this to the docs